public/ai-share/share.css

```diff
@@ -0,0 +1,194 @@
+:root {
+  --primary-color: #007bff;
+  --secondary-color: #6c757d;
+  --background-color: #f8f9fa;
+  --post-background-color: #ffffff;
+  --details-background-color: #e9ecef;
+  --details-hover-color: #dee2e6;
+}
+
+body {
+  font-family: Söhne, ui-sans-serif, system-ui, -apple-system, Segoe UI, Roboto, Ubuntu, Cantarell, Noto Sans, sans-serif, Helvetica Neue, Arial, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji;
```

Comment:
Why have so many fonts here?

Reply:
Should probably get designer Kris to take a pass with this CSS and page

Reply:
yeah going to hand this back to Kris tomorrow, for sure.